src/libponyc/codegen/genfun.c

```diff
@@ -609,6 +658,13 @@ void genfun_param_attrs(compile_t* c, reach_type_t* t, reach_method_t* m,
   reach_type_t* type = t;
   token_id cap = m->cap;
   int i = 0;
+  int offset = 1;
+
+  if(cap == TK_AT)
+  {
+    i = 1;
+    offset = 0;
+  }
```

Comment:
CI is complaining about this - either need to surround this block with `#if PONY_LLVM >= 309`, or add a `(void)offset;` statement to prevent compiler warnings/errors about this variable not being used when the `PONY_LLVM >= 309` condition on the later line is not met.